Clients/src/services/auth.api.js

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios';
+
+const API_URL = "http://127.0.0.1:8000/api";
```

Comment:
This can be a .env variable, so in the deployment server we won't need to edit this code file. And also in another dev computer as well.